.github/workflows/build_bazel.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 The StableHLO Authors.
```

Comment:
How about aligning names of the two build-related .yml files? E.g. `buildAndTestCMake.yml` and `buildBazel.yml`?

Reply:
+1